setup.py

```diff
@@ -59,7 +59,7 @@ def _dbt_trino_version():
         ]
     },
     install_requires=[
-        "dbt-core=={}".format(dbt_version),
+        "dbt-core~={}".format(dbt_version),
```

Comment:
yes! thank you for doing this, it should make things a lot easier going forward :)

Reply:
Just `1.1.x`. We're asking adapter maintainers to pin to minor versions, since there might be small changes in the adapter (Python) interface in new minor versions that require action. We'll be asking you to bump the version and re-test for each new minor version (once every three months). Those changes will never come in patch releases, though—those are fixes only.

Our longer-term goal is guaranteeing backwards compatibility for the adapter interface, so that you can pin to the major version instead (`1.x`)